src/main/java/com/epam/storage/TrainingTypeInitializer.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.storage;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Component
+public class TrainingTypeInitializer {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Transactional
+    public void initializeTrainingTypes() {
+        entityManager.createNativeQuery("INSERT INTO training_type (id, training_type_name) VALUES (1, 'Technical')")
+                .executeUpdate();
+        entityManager.createNativeQuery("INSERT INTO training_type (id, training_type_name) VALUES (2, 'Soft Skills')")
+                .executeUpdate();
+        entityManager.createNativeQuery("INSERT INTO training_type (id, training_type_name) VALUES (3, 'Leadership')")
+                .executeUpdate();
```

Comment:
Is it possible to insert all of this rows in one query?

Reply:
And maybe it's possible to have the autogenerated Id? Ping me about id autogeneration, if you need, I can provide a link to read.